code/clue.js

```diff
@@ -1,59 +1,361 @@
+    document.getElementById(loadID.id).style.opacity = 1;
+    setTimeout(function () {
+      document.getElementById(loadID.id).style.opacity = 0;
+      mystery.killer = randomSelector(suspects);
+      document.getElementById('killerName').style.background = mystery.killer.color;
+      document.getElementById('killerImage').src = mystery.killer.image;
+      document.getElementById('killerName').innerHTML = (`${mystery.killer.firstName} ${mystery.killer.lastName}`);
+      const weaponID = mystery.killer.favoriteWeapon;
+      const weaponName = weapons.find(weapon => weapon.id === weaponID).name;
+      document.getElementById('killerFavoriteWeapon').innerHTML = (`Preferred weapon: ${weaponName}`);
+      document.getElementById('killerAge').innerHTML = (`Age: ${mystery.killer.age}`);
+      document.getElementById('killerOccupation').innerHTML = (`Occupation: ${mystery.killer.occupation}`);
+      document.getElementById('killerDescription').innerHTML = (`Description: ${mystery.killer.description}`);
+      killerPicked = true;
+      checkScroll();
+    }, 2000);
+  }
+}
 
-// This function will be invoked when you click on the killer card.
-const pickKiller = () => {
-  // This will randomly select a killer from the suspects. And add that to the mystery object.
-  mystery.killer = randomSelector(suspects)
+const pickWeapon = (loadID) => {
+  if (killerPicked) {
+    const weaponsIncreased = increaseChanceForFavWeapon(mystery.killer);
+    document.getElementById(loadID.id).style.opacity = 1;
+    setTimeout(function () {
+      document.getElementById(loadID.id).style.opacity = 0;
+      mystery.weapon = randomSelector(weaponsIncreased);
+      document.getElementById('weaponName').style.background = mystery.weapon.color;
+      document.getElementById('weaponName').innerHTML = `Type: ${mystery.weapon.name}`;
+      document.getElementById('weaponWeight').innerHTML = `Weight: ${mystery.weapon.weight}`;
+      weaponPicked = true;
+      checkScroll();
+    }, 2000);
+  } else if (showInfoMessage("pickWeapon"));
+}
 
-  // This will change the background color of the card to the one connected to the chosen killer and show the full name of the killer. Feel free to add more things to show about the killer.
-  document.getElementById('killerCard').style.background = mystery.killer.color
-  document.getElementById(
-    'killerName'
-  ).innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`
+const pickRoom = (loadID) => {
+  if (killerPicked && weaponPicked) {
+    document.getElementById(loadID.id).style.opacity = 1;
+    setTimeout(function () {
+      document.getElementById(loadID.id).style.opacity = 0;
+      const increasedRoom = increaseChanceForRoom(mystery.weapon);
+      mystery.room = randomSelector(increasedRoom);
+      document.getElementById('roomName').innerHTML = mystery.room;
+      roomPicked = true;
+      checkScroll();
+    }, 2000);
+  } else(showInfoMessage("pickRoom"));
```

Comment:
This confused me a bit because I initially read it as 
```javascript
} else (showInfoMessage("pickRoom")) {
 ...
}
```
As if `showInfoMessage` should return a value, but it doesn't.
As @emeliesv mentioned, there is really no else.  Javascript will interpret this as 
```javascript 
} else (showInfoMessage("pickRoom")){
 ;
}
```
Which calls the function `showInfoMessage` when checking if the `else` is `true/false` but disregards the result.  The end result is the same since you are showing the message though.

Restructure it a bit to be more clear 👍